docs/content/how-to/aws/migrate-hosted-cluster.md

```diff
@@ -0,0 +1,990 @@
+BASE_PATH=${HOME}/hypershift
+BASE_DOMAIN="aws.sample.com"
+PULL_SECRET_FILE="${HOME}/pull_secret.json"
+AWS_CREDS="${HOME}/.aws/credentials"
+AWS_ZONE_ID="Z02718293M33QHDEQBROL"
+
+CONTROL_PLANE_AVAILABILITY_POLICY=SingleReplica
+HYPERSHIFT_PATH=${BASE_PATH}/src/hypershift
+HYPERSHIFT_CLI=${HYPERSHIFT_PATH}/bin/hypershift
+HYPERSHIFT_IMAGE=${HYPERSHIFT_IMAGE:-"quay.io/${USER}/hypershift:latest"}
+NODE_POOL_REPLICAS=${NODE_POOL_REPLICAS:-2}
+
+# MGMT Context
+MGMT_REGION=us-west-1
+MGMT_CLUSTER_NAME="${USER}-dev"
+MGMT_CLUSTER_NS=${USER}
+MGMT_CLUSTER_DIR="${BASE_PATH}/hosted_clusters/${MGMT_CLUSTER_NS}-${MGMT_CLUSTER_NAME}"
+MGMT_KUBECONFIG="${MGMT_CLUSTER_DIR}/kubeconfig"
+
+# MGMT2 Context
+MGMT2_CLUSTER_NAME="${USER}-dest"
+MGMT2_CLUSTER_NS=${USER}
+MGMT2_CLUSTER_DIR="${BASE_PATH}/hosted_clusters/${MGMT2_CLUSTER_NS}-${MGMT2_CLUSTER_NAME}"
+MGMT2_KUBECONFIG="${MGMT2_CLUSTER_DIR}/kubeconfig"
+
+# Hosted Cluster Context
+HC_CLUSTER_NS=clusters
+HC_REGION=us-west-1
+HC_CLUSTER_NAME="${USER}-hosted"
+HC_CLUSTER_DIR="${BASE_PATH}/hosted_clusters/${HC_CLUSTER_NS}-${HC_CLUSTER_NAME}"
+HC_KUBECONFIG="${HC_CLUSTER_DIR}/kubeconfig"
+BACKUP_DIR=${HC_CLUSTER_DIR}/backup
+
+BUCKET_NAME="${USER}-hosted-${MGMT_REGION}"
+```
+
+</details>
+
+And this is how the Migration workflow will happen
+
+![gif](../../images/hc-migration-workflow.gif)
+
+
+## Backup
+
+This section complains interaction among multiple components. We will need to backup all the relevant things to raise up this same cluster in our target management cluster.
+
+To do that we will: 
+
+1. Mark the Hosted Cluster with a ConfigMap which will declare the source Management Cluster it comes from (This is not mandatory but useful).
+
+<details>
+<summary>Config Map creation to set the Source Management Cluster</summary>
+
+
+```bash
+oc create configmap mgmt-parent-cluster -n default --from-literal=from=${MGMT_CLUSTER_NAME}
+```
+
+</details>
+
+2. Shutdown the reconciliation in the HostedCluster we want to migrate and also in the Nodepools.
+
+
+<details>
+<summary>ControlPlane Migration</summary>
+
+```bash
+PAUSED_UNTIL="true"
+oc patch -n ${HC_CLUSTER_NS} hostedclusters/${HC_CLUSTER_NAME} -p '{"spec":{"pausedUntil":"'${PAUSED_UNTIL}'"}}' --type=merge
+oc scale deployment -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} --replicas=0 kube-apiserver openshift-apiserver openshift-oauth-apiserver control-plane-operator
+```
+
+</details>
+
+<details>
+<summary>ControlPlane + NodePool Migration</summary>
+
+```bash
+PAUSED_UNTIL="true"
+oc patch -n ${HC_CLUSTER_NS} hostedclusters/${HC_CLUSTER_NAME} -p '{"spec":{"pausedUntil":"'${PAUSED_UNTIL}'"}}' --type=merge
+oc patch -n ${HC_CLUSTER_NS} nodepools/${NODEPOOLS} -p '{"spec":{"pausedUntil":"'${PAUSED_UNTIL}'"}}' --type=merge
+oc scale deployment -n ${HC_CLUSTER_NS}-${HC_CLUSTER_NAME} --replicas=0 kube-apiserver openshift-apiserver openshift-oauth-apiserver control-plane-operator
+```
+
+</details>
+
+3. Backup ETCD and Upload to S3 Bucket
```

Comment:
do we want to refer to https://deploy-preview-1780--hypershift-docs.netlify.app/how-to/aws/etc-backup-restore/ ?